forms/Form.php

```diff
@@ -486,19 +540,21 @@ public function handleField($request) {
 	public function makeReadonly() {
 		$this->transform(new ReadonlyTransformation());
 	}
-	
+
 	/**
-	 * Set whether the user should be redirected back down to the 
-	 * form on the page upon validation errors in the form or if 
+	 * Set whether the user should be redirected back down to the
+	 * form on the page upon validation errors in the form or if
 	 * they just need to redirect back to the page
 	 *
-	 * @param bool Redirect to the form
+	 * @param bool $bool
+	 * @return $this
+	 * @internal param Redirect $bool to the form
```

Comment:
So too late, but PHPStorm adds `@internal` when parameter names change. It could have been removed...